overlord/snapstate/handlers.go

```diff
@@ -1006,6 +1006,32 @@ func writeSeqFile(name string, snapst *SnapState) error {
 	return osutil.AtomicWriteFile(p, b, 0644, 0)
 }
 
+// ComputeMissingDisabledServices returns a list of services that were disabled
+// that are currently missing from the specific snap info (i.e. they were
+// renamed in this snap info), as well as a list of disabled services that are
+// present in this snap info
+// note that the union of the missing and found svc lists should be exactly
+// equal to st.LastActiveDisabledServices
+func ComputeMissingDisabledServices(st *SnapState, info *snap.Info) ([]string, []string, error) {
```

Comment:
just a couple of style remarks:

* st is always  *state.State,  we use snapst (or variations thereof) for *SnapState
* this should probably not be exported I imagine,  just `missingDisabledServices` without compute, and then it can export_test.go-ed for testing

Reply:
Okay noted for future.
* Changed to be only exported through export_test
* Also applied suggestion from https://github.com/snapcore/snapd/pull/7431#discussion_r336457370 here to use the list of services instead of SnapState